packages/opc-nav/webpack.config.js

```diff
@@ -0,0 +1,49 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+    mode: 'development',
+    devServer: {
+        contentBase: path.join(__dirname, 'dist'),
+        open: true,
+        compress: true,
+        port: 4200,
+    },
+    entry: `${path.join(__dirname, 'src/opc-nav')}`,
+    devtool: 'inline-source-map',
+    optimization : {
+        usedExports: true
+    },
+    module: {
+        rules: [
+        {
+            test: /\.tsx?$/,
+            use: 'ts-loader',
+            exclude: /node_modules/,
+        },
+        {
+            test: /\.css|\.s(c|a)ss$/,
+            use: [{
+              loader: 'lit-scss-loader',
+              options: {
+                minify: true, // defaults to false
+              },
+            }, 'extract-loader', 'css-loader', 'sass-loader'],
+        },
```

Comment:
@akhilmhdh we have a .scss loader here you can use scss file

Reply:
Changed